lib/Importer/Caffe2ModelLoader.cpp

```diff
@@ -214,7 +279,49 @@ void Caffe2ModelLoader::loadOperator(const caffe2::OperatorDef &op) {
     return;
   }
 
-  if (typeName == "MaxPool" || typeName == "AveragePool") {
+  if (typeName == "Int8SumRelu") {
```

Comment:
Can we make Caffe2 automatically unfuse this op for us? So we'd only support Int8Sum and Int8Relu?

Reply:
> Therefore, they don't really need "Int8Relu".

My understanding is that there is real range of numbers in the input activation. Let's say it is [-5.5, +5.5]. In Glow when we quantize it, we'll use the whole range of [-128, +127]. So our Relu will do `x' = max(x, 0)`. But in Caffe2 they use the whole range of [0, 255]. So their Relu will do `x' = max(x, 128)`. Essentially if input range is symmetric, then Relu needs to "cut" half of it. It's not possible to throw Relu away. Please correct me if I'm wrong.

> If we want the unfused op, the original fp32 model should be rewritten and re-quantized.

Why? You do unfuse here and keep the same quantization parameters.

> decomposing Int8SumRelu is not complicated

All decompositions are simple, but if we have too many of them, they could be maintenance burden.